pilot/pkg/features/pilot.go

```diff
@@ -281,7 +281,7 @@ var (
 
 	EnableVirtualServiceDelegate = env.RegisterBoolVar(
 		"PILOT_ENABLE_VIRTUAL_SERVICE_DELEGATE",
-		false,
+		true,
```

Comment:
I think test needs to be updated because the following condition will pass.

https://github.com/istio/istio/blob/7d99a5683d0764bae4194365dd6e420126885acd/pilot/pkg/model/virtualservice.go#L129-L133